assignment.rb

```diff
@@ -0,0 +1,69 @@
+require 'yaml'
+require 'byebug'
+require 'time'
+
+class User
+  # @@users_data = YAML.load(File.read('users_data.yaml'))
+  
+  def all_users
+    YAML.load_file 'users_data.yaml'
```

Comment:
Use memorization here otherwise it will load data every time we call this method.